procedures/src/main/java/com/intel/dai/procedures/ComputeNodeSetStates.java

```
@@ -100,6 +100,15 @@ public long run(String[] saNodeLctns, String sNodeNewState, long lTsInMicroSecs,
                 throw new VoltAbortException("Invalid state change was attempted from ERROR to ACTIVE");
             }
 
+            //----------------------------------------------------------------------
+            // Ensure that we do not transition the compute node from active to dhcp_discovered state
+            // (for a compute node if a node is in active it should not later directly transition to dhcp_discovered)!
+            //----------------------------------------------------------------------
+            if ((sNodeNewState.equals("D")) && (aNodeData[0].getString("State").equals("A"))) {
```

Instead of ignoring the DHCP requests message, you should raise a RAS event if the MAC address in DHCPDISCOVER message is not part of dai-ds configuration.

---

So EVERY networked equipment MUST be in our manifest? What if its not? RAS events will be fired that are not errors.